lib/jekyll-gist/gist_tag.rb

```diff
@@ -69,7 +70,7 @@ def fetch_raw_code(gist_id, filename = nil)
             read_timeout: 3, open_timeout: 3) do |http|
             request = Net::HTTP::Get.new uri.to_s
             response = http.request(request)
-            response.body
+            response.body.force_encoding(@encoding)
```

Comment:
Perhaps, but if the Jekyll user sets their site to use something else, and jekyll-gist returns utf-8, is Liquid not going to error out as it is now, with the mix of utf-8 and ascii 8bit?